solr/core/src/java/org/apache/solr/handler/component/HttpShardHandlerFactory.java

```diff
@@ -413,139 +384,4 @@ public void initializeMetrics(SolrMetricsContext parentContext, String scope) {
         solrMetricsContext.getMetricRegistry(),
         SolrMetricManager.mkName("httpShardExecutor", expandedScope, "threadPool"));
   }
-
```

Comment:
It's nice to see all this stuff moved out of HttpShardHandlerFactory which already has tons of concerns to deal with.